src/test/io/uuddlrlrba/ktalgs/math/NewtonMethodKtTest.kt

```diff
@@ -34,8 +35,8 @@ class NewtonMethodKtTest {
         assertEquals(2.toDouble(), sqrt(4.toDouble()), 1e-15)
         assertEquals(4.toDouble(), sqrt(16.toDouble()), 1e-12)
         for (i in 7 until 99) {
-            for (j in -15 until -1) {
-                assertEquals(sqrt(i.toDouble()), sqrt(i.toDouble()), j.toDouble())
+            for (j in 2 until 16) {
```

Comment:
This looks like a functional change, the test intended to validate negative numbers